cmake/build_helper.cmake

```diff
@@ -58,7 +58,7 @@ function(irm_add_arm_executable name)
         COMMAND st-flash --reset write ${BIN_FILE} 0x8000000
         DEPENDS ${name}.elf)
 
-    if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    if (NOT CMAKE_BUILD_TYPE STREQUAL "Release")
```

Comment:
Sanity check: are you changing the default cmake behavior to "DEBUG"?

Reply:
Nope. Adding another build type `RelWithDebInfo`, which generates debug symbols with full compiler optimizations enabled.